src/Domain/Regulation/Enum/VehicleTypeEnum.php

```diff
@@ -13,16 +13,13 @@ enum VehicleTypeEnum: string
     case PEDESTRIANS = 'pedestrians'; // Pedestrians.
     case BICYCLE = 'bicycle'; // Bicycle.
     case AMBULANCE = 'ambulance'; // A motor vehicle of category M intended for the transport of sick or injured people and having special equipment for such purpose.
-    case CRITAIR_4 = 'critair4'; // See: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000032749723/
-    case CRITAIR_5 = 'critair5'; // See: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000032749723/
     case OTHER = 'other';
 
     public static function restrictedCases(): array
     {
         return [
             VehicleTypeEnum::HEAVY_GOODS_VEHICLE,
-            VehicleTypeEnum::CRITAIR_4,
-            VehicleTypeEnum::CRITAIR_5,
+            CritairEnum::CRITAIR,
```

Comment:
Je pinaille sûrement, mais je dirais que `::CRITAIR` devrait se trouver sur `VehicleTypeEnum`

Comme ça on garde le fait que toutes les possibilités de boutons de types de véhicules concernés / exemptés sont sur `VehicleTypeEnum`

Et `CritairEnum` se concentre sur les choix possibles de vignette Crit'Air (il n'y a pas d'option "Crit'Air" sans numéro dans la liste de checkboxes)

```suggestion
            VehicleTypeEnum::CRITAIR,
```